src/potions.ts

```diff
@@ -35,13 +44,16 @@ import {
   Mood,
   sum,
   sumNumbers,
+  withChoice,
 } from "libram";
 import { acquire } from "./acquire";
 import { baseMeat, HIGHLIGHT, maxBy, pillkeeperOpportunityCost, turnsToNC } from "./lib";
 import { embezzlerCount } from "./embezzler";
 import { usingPurse } from "./outfit";
 import { estimatedGarboTurns } from "./turns";
 import { globalOptions } from "./config";
+import { bestShadowRift } from "./fights";
+import { chooseQuest, rufusTarget } from "libram/dist/resources/2023/ClosedCircuitPayphone";
```

Comment:
just import the module from libram instead of importing the individual functions from the libram module